cli.js

```diff
@@ -62,19 +65,43 @@ if (!tasks || Object.keys(tasks).length < 1) {
 	process.exit(0);
 }
 
+// get package.json descriptions value
+if (argv.descriptions) {
+	try {
+		descriptions = { exports: {} };
+		require("pkginfo")(descriptions, { dir: cwd, include: ["ntl"] });
+		descriptions = descriptions.exports.ntl.descriptions;
+	} catch (e) {
+		error(e, "No descriptions for your npm scripts found");
+	}
+}
+
 // defines the items that will be printed to the user
 const input = (argv.info
 	? Object.keys(tasks).map(i => ({ name: `${i}: ${tasks[i]}`, value: i }))
-	: Object.keys(tasks)
+	: argv.descriptions
+		? Object.keys(tasks).map(i => {
+			return {
+				name: `${i}: ${descriptions[i]}`,
+				value: i
+			}
+		})
+		: Object.keys(tasks)
```

Comment:
I think this can simply be:

```js
const input = (argv.info || argv.descriptions
 	? Object.keys(tasks).map(i => ({ name: `${i}: ${argv.descriptions ? descriptions[i] : tasks[i]}`, value: i }))
	: Object.keys(tasks)
```

This way descriptions also take precedence before "info" which I think would be more desirable